alibi/explainers/__init__.py

```diff
@@ -2,6 +2,7 @@
 The 'alibi.explainers' module includes feature importance, counterfactual and anchor-based explainers.
 """
 
+from .ale import ALE, plot_ale
```

Comment:
How do you mean? `plot_ale` takes an `Explanation` object, we had a discussion earlier how we with the setup we have now.

Reply:
My argument here is that the individual files will become a bit hard to navigate. Taking a naive example in SHAP, there are lots of different visualisations and each involve quite a bit of code. I think it can get very messy if we don't do something like `alibi.explainers.visualisation` and mirror the file structure. I think it would be easier to maintain, as the code would be easier to track down (e.g., for SHAP you don't need to scroll over the first 1000 lines of wrapper code etc)